bitnami/common/templates/_ingress.tpl

```diff
@@ -40,3 +40,14 @@ Usage:
 {{- print "true" -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Returns true if the ingressClassname field is supported
+Usage:
+{{ include "common.ingress.supportsIngressClassname" . }}
+*/}}
+{{- define "common.ingress.supportsIngressClassname" -}}
```

Comment:
I would make it in a similar way as the macro above. In order to avoid issues with the functions, I'm more into using strings instead of return `true` or `nothing` (ideally we should return `false` but it won't work)

Reply:
Indeed, that is the only helper in the whole repository that uses `print "true"`:

```
> grep -R 'print "true"' .                   
./charts/bitnami/common/templates/_ingress.tpl:{{- print "true" -}}
> grep -R 'print "false"' .
./charts/bitnami/common/templates/_ingress.tpl:{{- print "false" -}}
```